sklearn/utils/tests/test_themes.py

```diff
@@ -0,0 +1,235 @@
+import pytest
+
+from sklearn.utils import themes
+
+TEMPLATE_SIMPLE = """
+#$id {
+  color: $$color_1;
+  background-color: $$color_2;
+}
+""".replace("  ", "").replace("\n", "")  # noqa
+
+TEMPLATE_SIMPLE_2 = """
+#$id {
+  color: $$color_1;
+}
+#$id div {
+  background-color: $$color_2;
+}
+""".replace("  ", "").replace("\n", "")  # noqa
+
+
+def test_original_style():
+    # Test that themes.LIGHT matches up with the original stylesheet.
+    # This test can be deleted when themes.LIGHT style changes.
+    _STYLE = """
```

Comment:
This type of testing is basically testing that `string.Template` works. It also makes it harder to maintain because any update to the original style requires an update here. I prefer not to not have this test.

Reply:
My original intention was to show that the there was no changes to the original `_STYLE` stylesheet after going through the theme/template builder.

Now with CSS variables approach, the original `_STYLE` stylesheet will be changed, so won't need it. Have removed it